security/acls/acl.sh

```diff
@@ -48,6 +48,13 @@ check_timeout || exit 1
 check_mvn || exit 1
 check_expect || exit 1
 
+REQUIRED_CCLOUD_VER="0.185.0"
```

Comment:
@rspurgeon excellent addition!  There are other demos that may need this so it may be more valuable to move this entire code segment directly INTO https://github.com/confluentinc/examples/blob/5.3.1-post/utils/helper.sh#L50-L60 .  WDYT?

Reply:
See code function above on what it could look like: https://github.com/confluentinc/examples/blob/5.3.1-post/utils/helper.sh#L36-L48

Reply:
I added a fuller function but left a parameter w/ default in, in case a caller wants to check a against a different version